src/FileFormatConverter.h

```diff
@@ -0,0 +1,322 @@
+/*
+ * Souffle - A Datalog Compiler
+ * Copyright (c) 2018, The Souffle Developers. All rights reserved
+ * Licensed under the Universal Permissive License v 1.0 as shown at:
+ * - https://opensource.org/licenses/UPL
+ * - <souffle root>/licenses/SOUFFLE-UPL.txt
+ */
+
+/************************************************************************
+ *
+ * @file FileFormatConverter.h
+ *
+ * Defines a class providing static methods for file format conversion.
+ *
+ ***********************************************************************/
+
+#pragma once
+
+#include <cassert>
+#include <fstream>
+#include <functional>
+#include <map>
+#include <set>
+#include <sstream>
+#include <stdexcept>
+#include <string>
+#include <vector>
+
+namespace souffle {
```

Comment:
If this is moved to profilerlib, don't use the souffle namespace (unless the whole thing does)